src/docbuilder/rustwide_builder.rs

```diff
@@ -95,12 +96,43 @@ impl RustwideBuilder {
         // Ignore errors if detection fails.
         let old_version = self.detect_rustc_version().ok();
 
+        let mut targets_to_install = TARGETS
+            .iter()
+            .map(|t| t.to_string())
+            .collect::<HashSet<_>>();
```

Comment:
Since we don't need to modify this (see above), I don't think we need a second data structure, we can just use `.contains()` on TARGETS on line 124.